SPECS/grpc/grpc.spec

```diff
@@ -98,7 +98,6 @@ popd
 %install
 pushd cmake/build
 %cmake_install
-find %{buildroot} -name '*.cmake' -delete
```

Comment:
@neha170 It's been a while, but do you remember why you excluded these files from packaging when writing the original spec? No worries if not, just want to make sure there isn't something else we need to address before taking this change.

Reply:
@oliviacrain unfortunately, I don't remember at all. I am guessing the linter suggested this.

Reply:
I don't believe that's the case- the linter only corrects already-present `*.la` removal commands to use a standardized command.

Thanks for taking a look! Resolving, since it looks like there's no concerns over merging this.